next.config.js

```diff
@@ -7,6 +7,16 @@ const withNextIntl = createNextIntlPlugin();
 const nextConfig = {
     output: 'standalone', // Outputs a Single-Page Application (SPA).
     distDir: './dist', // Changes the build output directory to `./dist/`.
+    experimental: {
+        turbo: {
+            rules: {
+                '*.svg': {
+                    loaders: ['@svgr/webpack'],
```

Comment:
The Issue with turbopack ist, that our webpack config below doesn't apply. And I couldn't find a way to set the rules for the *.svg?url....